src/core/proxy.ts

```diff
@@ -74,7 +72,9 @@ export function createProxyProxy<T extends Objectish>(
 		copy_: null,
 		// Called by the `produce` function.
 		revoke_: null as any,
-		isManual_: false
+		isManual_: false,
+		// `callbacks` actually gets assigned in `createProxy`
+		callbacks_: []
```

Comment:
Q: would it help to pre-initialize a size? this captures all modifications for a single immer run? maybe 100 to avoid re-allocations on small recipes?